04-conditional-logic/factors.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+def get_a_positive_integer_from_user():
+    response = int(input("Enter a positive integer: "))
+
+    if not response:
+        return get_a_positive_integer_from_user()
+
+    return response
+
+def find_divisor_from_integer(integer):
```

Comment:
You didn't respond to the comment about the function name:

> Additionally, this function does something different than its title suggest. It prints all the divisors of the given integer rather than find a divisor.

Please rename the function.